app/src/main/java/com/example/myapplication/ThirdFragment.java

```diff
@@ -36,35 +43,150 @@ public class ThirdFragment extends Fragment {
+        animation.setDuration(1000);
+        binding.pop.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                animation.start();
+                MainActivity.darkMode = !MainActivity.darkMode;
+                lightDark(v, MainActivity.darkMode);
+            }
+        });
+        binding.switch1.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
+            @Override
+            public void onCheckedChanged(CompoundButton buttonView, boolean isChecked)  {
+                if (binding.switch1.isChecked()){
+                    binding.switch1.setThumbTintList(purple);
+                    binding.switch1.setTrackTintList(purple);
+                } else {
+                    binding.switch1.setThumbTintList(blue);
+                    binding.switch1.setTrackTintList(blue);
+                }
+                MainActivity.broke = binding.switch1.isChecked();
+            }
+        });
+        binding.switch2.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
+            @Override
+            public void onCheckedChanged(CompoundButton buttonView, boolean isChecked)  {
+                if (binding.switch2.isChecked()){
+                    binding.switch2.setThumbTintList(purple);
+                    binding.switch2.setTrackTintList(purple);
+                } else {
+                    binding.switch2.setThumbTintList(blue);
+                    binding.switch2.setTrackTintList(blue);
+                }
+                MainActivity.broke = binding.switch2.isChecked();
+            }
+        });
+        binding.switch3.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
+            @Override
+            public void onCheckedChanged(CompoundButton buttonView, boolean isChecked)  {
+                if (binding.switch3.isChecked()){
+                    binding.switch3.setThumbTintList(purple);
+                    binding.switch3.setTrackTintList(purple);
+                } else {
+                    binding.switch3.setThumbTintList(blue);
+                    binding.switch3.setTrackTintList(blue);
+                }
+                MainActivity.broke = binding.switch3.isChecked();
+            }
+        });
         String[] typeContact = {"No contact", "Teammate to Teammate", "Teammate to opponent"};
         ArrayAdapter<String> contact = new ArrayAdapter<>(view.getContext(), android.R.layout.simple_spinner_dropdown_item, typeContact);
         binding.spinner.setAdapter(contact);
 
 
         Editable numNotes = binding.numNotes.getText();
 
+        lightDark(v, MainActivity.darkMode);
 
-        }
+        Switch switch1 = view.findViewById(R.id.switch1);
+        switch1.isChecked();
 
+        Switch switch2 = view.findViewById(R.id.switch2);
+        switch2.isChecked();
+
+        Switch switch3 = view.findViewById(R.id.switch3);
+        switch3.isChecked();
```

Comment:
Are those lines necessary? It seems like they are only retrieving the switch elements and getting their checked state, but not doing anything with this information.

Reply:
yes they will be required for data transfer